lib/redis_memo/future.rb

```diff
@@ -37,15 +37,23 @@ def method_cache_key
       RedisMemo::MemoizeMethod.__send__(:method_cache_keys, [context])&.first || ''
   end
 
+  def validate_cache_result
```

Comment:
do we need rdocs for `future.rb`? also it seems we don't have any validations on what are passed to `RedisMemo::DefaultOptions`, right?

Reply:
> do we need rdocs for future.rb?

People won't be interacting with future.rb directly too much, but it definitely  won't hurt with some rdocs :)

> we don't have any validations on what are passed to RedisMemo::DefaultOptions, right?

Yeah, if people pass in weird things, things would break. I think documenting what are the expected input is good enough -- we don't have to have runtime enforcement to make sure the input user provided is expected